redpanda/values.yaml

```diff
@@ -228,7 +228,7 @@ statefulset:
   replicas: 3
   updateStrategy:
     type: RollingUpdate
-  podManagementPolicy: Parallel
+  podManagementPolicy: OrderedReady
```

Comment:
Did you notice any problems that led you to make this change? Do you see any differences in how brokers are aligned with nodes? Just interested in why this change now, and I do think it logically makes sense.